src/rviz/visualization_frame.cpp

```diff
@@ -382,9 +382,11 @@ void VisualizationFrame::initialize(const QString& display_config_file)
 
 void VisualizationFrame::initConfigs()
 {
-  home_dir_ = QDir::toNativeSeparators(QDir::homePath()).toStdString();
+  std::wstring home_dir_wstring = QDir::toNativeSeparators(QDir::homePath()).toStdWString();
```

Comment:
Curious is it the same issue talking about [here](https://stackoverflow.com/questions/35373591/qts-qdir-file-names-dropping-non-ascii-characters)?

If yes, maybe it is worth trying to configure the process-wide setting at Rviz startup by calling `QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"))`. If it works (hopefully), then all the string conversation would benefit from the fix.

Reply:
Yes I think its related. Should this be set in the main.cpp of rviz?

Reply:
Before we push this change, can you verify @seanyen's question?

Reply:
Yes I tested it out by placing "QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"))" in main, but it did not solve the issue.